codestyle/checkstyle.xml

```diff
@@ -71,10 +71,31 @@
       <property name="illegalPattern" value="true"/>
       <property name="message" value="Use Comparators.naturalNullsFirst() instead of Ordering.natural().nullsFirst()"/>
     </module>
+
     <module name="Regexp">
       <property name="format" value="(Byte|Character|Short|Integer|Long|Float|Double)\.TYPE"/>
       <property name="illegalPattern" value="true"/>
       <property name="message" value="Use primitive.class instead. But check twice that you don't actually need BoxedPrimitive.class instead of BoxedPrimitive.TYPE"/>
     </module>
+    <module name="Regexp">
+      <property name="format" value="Float\.MAX_VALUE"/>
+      <property name="illegalPattern" value="true"/>
+      <property name="message" value="Use Float.POSITIVE_INFINITY"/>
+    </module>
+    <module name="Regexp">
+      <property name="format" value="Float\.MIN_VALUE"/>
```

Comment:
not sure how this check style will work. Does this means any usage of `Float.MIN_VALUE` is prohibited ?